:lipstick: Refactor and update stylings to correct for conflicts in the new repo

Erik-Martus opened the pull request:
I made changes to stylings to the Hosting Path up to the create game lobby and on the Player Path up to the avatar picker. Style changes consisted mainly of making sure the classes matched the new HTML structure. Major changes were that colors and font-families pretty much exclusively use CSS variables in order to create consistency.

Erik-Martus committed:
:recycle: Update font file location

Updated location of font folder in repo (now under public instead of public/css)
Avenir Next now works

b7s9 committed:
✨ Display joined players on host
♻️ Refactor start button to only be on host
♻️ Add updated markup for early views
🔨 Remove textFits and console logs

Erik-Martus committed:
:recycle: Fix formatting

Fix indentation, remove unnecessary white space

Erik-Martus committed:
:recycle: Fix button styles

Erik-Martus committed:
:recycle: Refactor CSS files

Refactor based on changes contributed by Arpit in old repo

Erik-Martus committed:
:recycle: Refactor height styling

Erik-Martus committed:
:recycle: Cleanup white space

Erik-Martus committed:
:recycle: Remove unhelpful comments

Erik-Martus committed:
:recycle: Remove px sizing

b7s9 committed:
✨ Add markup to pl-prompt and waiting views

Erik-Martus committed:
:recycle: Refactor color usage to css variables

Erik-Martus committed:
:recycle: Refactor font-family implementation to use css variables

Erik-Martus committed:
:sparkles: Add overrides.css

Contains styles that override styles added by the system via JS

Erik-Martus committed:
:lipstick: Update create game stylings

Erik-Martus committed:
:recycle: Remove unused stylings

Removed stylings commented out from the alpha static

Erik-Martus committed:
:sparkles: Add avatar-picker.css

Erik-Martus committed:
:lipstick: Update text and input stylings

b7s9 reviewed (changes requested):
🐛 Create and Join buttons are partially hidden in the DOM. give a higher z-index or fix the obscuring element
💄 Closed captions should be hidden by default, I think?
💩 Uses deprecated HTML. Please update to newest markup version

b7s9 commented:
💄 System status error on avatar picker. have some kind of "selected" psuedo class or something

b7s9 committed:
Merge branch 'master' into front-end

bam32099 committed:
:lipstick: Room Code/Thunk Logo Changes

Room code circle now sits in corner as it is in the figma doc

Thunk Logo Font Issues are here!
(If theres an issue, its probably this one)

Erik-Martus committed:
:bug: Fix avenir overriding gilbert for thunk title

bam32099 committed:
:lipstick: Mobile Buttons Centered / Removed List Bullets

Fixed list styles so buttons look like buttons and mobile screen buttons are now centered, as they should be.

Spartacau5 committed:
:lipstick: assigned changes

Erik-Martus committed:
:books: Update git ignore for png files

Erik-Martus committed:
:lipstick: Update credits button positioning

Erik-Martus committed:
:sparkles: Add timer sizing and animation

Animation temporarily triggered on timer hover

bam32099 committed:
:lipstick: Scores - Master

Began updating list stylings.  Will include avatar images in next commit.

bam32099 committed:
Merge branch 'front-end' of https://github.com/project-dali/thunk-2 into front-end

Commit:
:lipstick: Banner, credits, scrolling

Changed banner from png to svg
Updated credits screen
Fixed scrolling

Commit:
:lipstick: Grids

Grids for response screen
